pwiz_tools/Skyline/CommandLine.cs

```diff
@@ -1717,6 +1723,34 @@ public void RemoveResults(DateTime? removeBefore)
             }
         }
 
+        public bool MinimizeResults(CommandArgs commandArgs)
+        {
+            if (!_doc.Settings.HasResults)
+            {
+                _out.WriteLine(Resources.CommandLine_ReintegratePeaks_Error__You_must_first_import_results_into_the_document_before_reintegrating_);
+                return false;
+            }
+
+            var saveFile = commandArgs.MinimizeFilePath ?? _skylineFile;
+            _out.WriteLine("Minimizing results to {0}", saveFile);
+            if (commandArgs.ChromatogramsDiscard)
+                _out.WriteLine("Removing unused chromatograms" + @"...");

```

Comment:
Why + "..."? An elipsis is localizable text. It is a single character in Chinese.

Reply:
I saw this line in RemoveResults and thought it was convention:
Resources.CommandLine_RemoveResults_Removing_results_before_ + removeBefore.Value.ToShortDateString() + @"..."

Should I change it here too?